src/main/java/com/cloudbees/jenkins/support/api/FilePathContent.java

```diff
@@ -24,11 +24,11 @@
 
 package com.cloudbees.jenkins.support.api;
 
+import com.cloudbees.jenkins.support.SupportLogFormatter;
 import hudson.FilePath;
-import hudson.util.IOException2;
 
-import java.io.IOException;
-import java.io.OutputStream;
+import java.io.*;
```

Comment:
hmmm

Reply:
My IntelliJ settings might be a bit aggressive about wildcard imports. Didn't know if there is some standard regarding this. Is there one?

Reply:
We've been advising to use explicit imports since it makes having class name conflicts easier to resolve.